ksqldb-serde/src/main/java/io/confluent/ksql/serde/connect/ConnectKsqlSchemaTranslator.java

```diff
@@ -32,42 +32,42 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public final class ConnectSchemaUtil {
+public final class ConnectKsqlSchemaTranslator {
 
   private static final Logger log = LoggerFactory.getLogger(ConnectSchemaTranslator.class);
 
   private static final SqlSchemaFormatter FORMATTER =
       new SqlSchemaFormatter(w -> false, Option.AS_COLUMN_LIST);
 
-  private static final Map<Type, Function<Schema, Schema>> CONNECT_TO_KSQL =
-      ImmutableMap.<Type, Function<Schema, Schema>>builder()
-      .put(Type.INT8, s -> Schema.OPTIONAL_INT32_SCHEMA)
-      .put(Type.INT16, s -> Schema.OPTIONAL_INT32_SCHEMA)
-      .put(Type.INT32, ConnectSchemaUtil::convertInt32Schema)
-      .put(Type.INT64, ConnectSchemaUtil::convertInt64Schema)
-      .put(Type.FLOAT32, s -> Schema.OPTIONAL_FLOAT64_SCHEMA)
-      .put(Type.FLOAT64, s -> Schema.OPTIONAL_FLOAT64_SCHEMA)
-      .put(Type.STRING, s -> Schema.OPTIONAL_STRING_SCHEMA)
-      .put(Type.BOOLEAN, s -> Schema.OPTIONAL_BOOLEAN_SCHEMA)
-      .put(Type.BYTES, ConnectSchemaUtil::toKsqlBytesSchema)
-      .put(Type.ARRAY, ConnectSchemaUtil::toKsqlArraySchema)
-      .put(Type.MAP, ConnectSchemaUtil::toKsqlMapSchema)
-      .put(Type.STRUCT, ConnectSchemaUtil::toKsqlStructSchema)
-      .build();
+  private final Map<Type, BiFunction<ConnectKsqlSchemaTranslator, Schema, Schema>> connectToSql =
```

Comment:
Why is this changing to become non-static? (Under what scenarios will this mapping depend on the instance?)

Reply:
Because some methods like `toKsqlBytesSchema` are changed to non-static, to access those methods here, this map also needs to be non-static.